suggestionBox/suggestionBox-library/src/main/java/org/silverpeas/components/suggestionbox/model/SuggestionCollection.java

```diff
@@ -143,7 +142,7 @@ public boolean containsAll(final Collection<?> suggestions) {
 
   @Override
   public boolean addAll(final Collection<? extends Suggestion> suggestions) {
-    Transaction.performInOne(new Transaction.Process<Void>() {
+    Transaction.performInOne(new Process<Void>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1604)